### PR TITLE
Feature/sticky footer

### PR DESCRIPTION
**Pull request by dzmitryaleinik**
### Where

* **Pivotal Story:** _
https://coverwallet.atlassian.net/browse/D2A-582

### Screenshots

_If this PR is UI related provide related screenshots to your changes_
image

### CODE REVIEW

File: examples/basic/app.js [hunk 1]
```diff
@@ -501,6 +503,13 @@ const App = function App() {
           />
         </div>
       </div>
+      
+    </div>
+    <StickyFooter>
+        <div className="test">
```
Comment (arzii) [inline examples/basic/app.js]: `className="test"`

Comment (dzmitryaleinik) [reply examples/basic/app.js]: I used it just as example, want do delete after pr will approved

File: lib/stylesheets/components/sticky-footer.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  background-color: white;
+  color: $cwColor-black-xmedium;
+  width: 100%;
+  box-shadow: 1px -3px 5px 0 rgba(110, 110, 110, 0.14);
+  border: solid 1px #f0f0f0;
```
Comment (nadzeya-baharodava-itechart) [inline lib/stylesheets/components/sticky-footer.scss]: do we have $cwColor for that #f0f0f0 ?

File: lib/stylesheets/components/sticky-footer.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  font-family: 'OpenSans';
+  font-size: 15px;
+  z-index: 2;
+
+  @include media ($mobile) {
+    box-shadow: none;
+    background-color: #f9f9f9;
```
Comment (nadzeya-baharodava-itechart) [inline lib/stylesheets/components/sticky-footer.scss]: do we have $cwColor for that #f9f9f9 ?

File: lib/stylesheets/components/sticky-footer.scss (hunk 2 above)
Comment (arzii) [reply lib/stylesheets/components/sticky-footer.scss]: @nadzeya-baharodava-itechart what about `rgba(110, 110, 110, 0.14)`? :D

Comment (nadzeya-baharodava-itechart) [reply lib/stylesheets/components/sticky-footer.scss]: sorry, missed it :)

File: lib/stylesheets/components/sticky-footer.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  bottom: 0;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  background-color: white;
+  color: $cwColor-black-xmedium;
+  width: 100%;
+  box-shadow: 1px -3px 5px 0 rgba(110, 110, 110, 0.14);
+  border: solid 1px #f0f0f0;
+  font-family: 'OpenSans';
```
Comment (yury-yaroshyk-itechart) [inline lib/stylesheets/components/sticky-footer.scss]: I don't think we need this font-family for footer. each application has default font and we need to inherit it. imho

Comment (arzii) [reply lib/stylesheets/components/sticky-footer.scss]: Agree with @yury-yaroshyk-itechart, additionally when you define font-family don't forget about fallback font like `sans-serif`.

Comment (dzmitryaleinik) [reply lib/stylesheets/components/sticky-footer.scss]: Agree with that

File: lib/stylesheets/components/sticky-footer.scss (hunk 3 above)
Comment (dzmitryaleinik) [reply lib/stylesheets/components/sticky-footer.scss]: No, we don't have here, but we do in my-coverwallet, so i will add it.

File: lib/stylesheets/components/sticky-footer.scss (hunk 4 above)
Comment (dzmitryaleinik) [reply lib/stylesheets/components/sticky-footer.scss]: Agree with that

Comment (dzmitryaleinik) [reply lib/stylesheets/components/sticky-footer.scss]: Agree with that

File: lib/stylesheets/components/sticky-footer.scss (hunk 3 above)
Comment (dzmitryaleinik) [reply lib/stylesheets/components/sticky-footer.scss]: No, we don't have here, but we do in my-coverwallet, so i will add it.

Comment (dzmitryaleinik) [reply lib/stylesheets/components/sticky-footer.scss]: no, we don't, but we do have in my-coverwallet, so i will add it

Comment (dzmitryaleinik) [reply lib/stylesheets/components/sticky-footer.scss]: no, we don't, but we do have in my-coverwallet, so i will add it

File: lib/stylesheets/components/sticky-footer.scss (hunk 4 above)
Comment (dzmitryaleinik) [reply lib/stylesheets/components/sticky-footer.scss]: agree with that

**Pull request merged**